test_opensearchpy/test_async/test_server/test_plugins/__init__.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
```

Comment:
This is a new file, so I believe everything including and down from Modifications ... should be removed. Just leave the SPDX header and the paragraph that says "The OpenSearch ... open source license.".